service/internal/templates.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"html/template"
+	"io"
+	"io/ioutil"
+	"log"
+)
+
+var (
+	fs                = FS(false)
```

Comment:
You cannot call with "true" from production code, because the file will not be present in the final binary. So need to call from the embedded data. I think this does not require comment, only tests should use `true`.